core/client/gobindClient/client.go

```diff
@@ -0,0 +1,174 @@
+// TODO(amarcedone) : rename package to gobind_client after https://github.com/golang/go/issues/17359 is solved.
+package gobindClient
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/keytransparency/cmd/keytransparency-client/grpcc"
+	"github.com/google/trillian"
+	"github.com/google/trillian/client"
+	"github.com/google/trillian/crypto/keys/pem"
+	"github.com/google/trillian/merkle/hashers"
+	_ "github.com/google/trillian/merkle/objhasher" // Used to init the package so that the hasher gets registered (needed by the bGetVerifierFromParams function)
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"net"
+	"time"
+
+
+	"crypto/tls"
+	"crypto/x509"
+	"github.com/google/keytransparency/core/client/kt"
+	"github.com/google/keytransparency/core/crypto/keymaster"
+	"github.com/google/keytransparency/core/crypto/vrf/p256"
+	pb "github.com/google/keytransparency/impl/proto/keytransparency_v1_service"
+	"log"
+)
+
+type BClientParams struct {
+	KtURL        string
+	MapID        int64
+	KtTLSCertPEM []byte
+	VrfPubPEM    []byte
+	KtSigPubKey  []byte
+	LogPEM       []byte
+}
+
+// TODO(amarcedone) consider persisting the client or at least the trusted smr, to gain efficiency and stronger consistency guarantees.
```

Comment:
Indeed, the design is for the client to persist the trusted SMR itself. 
Please create a place for it. Later on we can store and retrieve it from storage etc.